js/modules/k6/http/request.go

```diff
@@ -56,47 +56,57 @@ var ErrHTTPForbiddenInInitContext = common.NewInitContextError("Making http requ
 // ErrBatchForbiddenInInitContext is used when batch was made in the init context
 var ErrBatchForbiddenInInitContext = common.NewInitContextError("Using batch in the init context is not supported")
 
-type HTTPRequest struct {
+// Request represent an http request
+type Request struct {
 	Method  string                          `json:"method"`
 	URL     string                          `json:"url"`
 	Headers map[string][]string             `json:"headers"`
 	Body    string                          `json:"body"`
 	Cookies map[string][]*HTTPRequestCookie `json:"cookies"`
 }
 
-func (h *HTTP) Get(ctx context.Context, url goja.Value, args ...goja.Value) (*HTTPResponse, error) {
+// Get makes a get request and returns a coresponsin response by taking goja.Values as arguments
```

Comment:
typo: `coresponsin` should be `corresponding` - both here and in the other comments below
also, `a get request` should probably be `an HTTP GET request`